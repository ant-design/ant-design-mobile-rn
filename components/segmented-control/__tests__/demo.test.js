import rnDemoTest from '../../../tests/shared/demoTest'

rnDemoTest('segmented-control')
