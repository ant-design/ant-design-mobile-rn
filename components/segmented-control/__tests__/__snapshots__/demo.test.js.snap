// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/segmented-control/demo/basic.tsx correctly 1`] = `
<View
  style={
    Object {
      "paddingHorizontal": 20,
      "paddingTop": 60,
    }
  }
>
  <Text>
    Disabled
  </Text>
  <RCTSegmentedControl
    enabled={false}
    onChange={[Function]}
    selectedIndex={0}
    style={
      Array [
        Object {
          "height": 28,
        },
        undefined,
      ]
    }
    tintColor="#108ee9"
    values={
      Array [
        "Segment1",
        "Segment2",
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 15,
        },
        undefined,
      ]
    }
  />
  <Text>
    TintColor and Style
  </Text>
  <RCTSegmentedControl
    enabled={true}
    onChange={[Function]}
    selectedIndex={0}
    style={
      Array [
        Object {
          "height": 28,
        },
        Object {
          "height": 40,
          "width": 280,
        },
      ]
    }
    tintColor="#ff0000"
    values={
      Array [
        "Segment1",
        "Segment2",
        "Segment3",
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 15,
        },
        undefined,
      ]
    }
  />
  <Text>
    SelectedIndex
  </Text>
  <RCTSegmentedControl
    enabled={true}
    onChange={[Function]}
    selectedIndex={1}
    style={
      Array [
        Object {
          "height": 28,
        },
        undefined,
      ]
    }
    tintColor="#108ee9"
    values={
      Array [
        "Segment1",
        "Segment2",
        "Segment3",
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 15,
        },
        undefined,
      ]
    }
  />
  <Text>
    onChange/onValueChange
  </Text>
  <RCTSegmentedControl
    enabled={true}
    onChange={[Function]}
    selectedIndex={0}
    style={
      Array [
        Object {
          "height": 28,
        },
        undefined,
      ]
    }
    tintColor="#108ee9"
    values={
      Array [
        "Segment1",
        "Segment2",
        "Segment3",
      ]
    }
  />
</View>
`;
